modules/nf-core/limma/differential/main.nf

```diff
@@ -4,17 +4,19 @@ process LIMMA_DIFFERENTIAL {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/bioconductor-limma:3.54.0--r42hc0cfd56_0' :
-        'biocontainers/bioconductor-limma:3.54.0--r42hc0cfd56_0' }"
+        'oras://community.wave.seqera.io/library/bioconductor-edger_bioconductor-ihw_bioconductor-limma_r-dplyr_r-readr:7fc48564d286c1c6' :
+        'community.wave.seqera.io/library/bioconductor-edger_bioconductor-ihw_bioconductor-limma_r-dplyr_r-readr:edea0f9fbaeba3a0' }"
```

Comment:
```suggestion
        'nf-core/bioconductor-edger_bioconductor-ihw_bioconductor-limma_r-dplyr_r-readr:edea0f9fbaeba3a0' }"
```
I pulled and pushed to quay.io

So we can change the registry from all container in a simpler way.
We'll be using `community.wave.seqera.io` as registry when we do the switch

Reply:
I'm not keen on this @maxulysse, there's no way for someone without privilege to do this and maintain the module.